editor/editor_themes.cpp

```diff
@@ -1404,6 +1406,8 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_stylebox("tab_hovered", "TabBar", style_tab_hovered);
 	theme->set_stylebox("tab_unselected", "TabBar", style_tab_unselected);
 	theme->set_stylebox("tab_disabled", "TabBar", style_tab_disabled);
+	theme->set_stylebox("tab_focus", "TabBar", style_tab_focus);
+	theme->set_stylebox("tab_focus", "TabContainer", style_tab_focus);
```

Comment:
The one for `TabContainer` should be grouped with other `TabContainer` styleboxes above.